tag_processor_plugin/tag_processor_plugin.go

```
@@ -0,0 +1,582 @@
+┌ Payload:
+%s
+
+└ Metadata:
+%s
+
+To fix: Set required fields (msg.meta.location_path, msg.meta.data_contract, msg.meta.tag_name) in defaults, conditions, or advancedProcessing.`,
+			strings.Join(missingFields, ", "),
+			payloadStr,
+			string(metadataJSON))
+
+		p.logError(fmt.Errorf(errorMsg), "metadata validation", msg)
```

golangci-lint (1.62.2)

printf: non-constant format string in call to fmt.Errorf

(govet)